### PR TITLE
Add Netty support

### PR DESCRIPTION
**Pull request by trask**
This also brings WebFlux support.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -55,6 +55,7 @@ include ':test:smoke:appServers:Tomcat.8.5'
 include ':test:smoke:appServers:JBossEAP.7'
 include ':test:smoke:appServers:JBossEAP.6'
 include ':test:smoke:appServers:Jetty.9'
+include ':test:smoke:appServers:JavaSE'
```
Comment (littleaj) [inline settings.gradle]: nice!

File: test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java [hunk 2]
```diff
@@ -553,26 +563,30 @@ private static void startTestApplicationContainer() throws Exception {
 
         final ContainerInfo containerInfo = new ContainerInfo(containerId, currentImageName);
         currentContainerInfo.set(containerInfo);
-        try {
-            String url = String.format("http://localhost:%s/", String.valueOf(appServerPort));
-            System.out.printf("Verifying appserver has started (%s)...%n", url);
+        if (currentImageName.startsWith("javase_")) {
```
Comment (littleaj) [inline test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java]: what's the rationale here?

**Comment by littleaj** [comment]
@trask, this PR resolves #790. We need to remove that line from the docs also :)

File: test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java (hunk 2 above)
Comment (trask) [reply test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java]: 👍 Good point, I was thinking with Java SE app there would not be an app server to handle the url, but that's not the case.

Comment (trask) [reply test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java]: I forgot why 😭. There was a reason. I added comment to explain.

**Comment by trask** [comment]
Since this is not going into master, I'm not sure if we should mark it as resolving #790.

**Pull request merged**